src/CalcViewModel/Snapshots.h

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+#include <vector>
+#include <memory>
+
+#include "CalcManager/CalculatorManager.h"
+
+namespace CalculatorApp::ViewModel::Snapshot
+{
+public
+    interface struct ICalcManagerIExprCommand
+    {
+    };
+
+public
+    ref struct UnaryCommand sealed : public ICalcManagerIExprCommand
+    {
+        property Windows::Foundation::Collections::IVectorView<int> ^ Commands {
+            Windows::Foundation::Collections::IVectorView<int> ^ get();
+            void set(Windows::Foundation::Collections::IVectorView<int> ^ commands);
+        };
+
+        UnaryCommand();
+
+        internal :;
```

Comment:
What's the meaning of this?

Reply:
`internal` is a special access specifier in C++/CX only. It means its following members are not going to be exported.
Adding an immediate semi-colon `;` here is a workaround to make the code friendly to clang-format since C++/CX is not under its support. Otherwise, after running clang-format on this source file, the lines around specifier `internal` are aweful :(